doc-content/enterprise-only/project-data/variables-con.adoc

```diff
@@ -19,6 +19,63 @@ If the variable cannot be found directly in the element's context, the immediate
 If the variable cannot be found, a read access request returns `null` and a write access produces an error message, and the process continues its execution.
 Variables are searched for based on their ID.
 
+== Variable tags
```

Comment:
@hmanwani-rh  this is new content right? So you need to create a new module.  The new file would look something like this:

[id="con-variable-tags_{context}"]

= Variable tags

For greater....

...

The file would be con-variables-tags.adoc.

Then you put the include statement in the master.htm.

Reply:
The variables-con.adoc has multiple sub-sections (== Defining global variables, == Defining process variables, == Defining local variables), therefore I added the Variable tags in the same file to follow the same. However, it makes sense to create the modules, therefore I am creating the procedure modules for all the sub-sections that are available in the Variables chapter, for example, Defining global variables, Defining process variables, Defining local variables. along with the conceptual module for Variable tags.